subprojects/docs/src/docs/userguide/console.adoc

```diff
@@ -121,8 +121,6 @@ Gradle provides a fined-grained view of the actual work being performed directly
 </figure>
 +++++
 
-CAUTION: Some work executed in parallel in not displayed yet e.g. parallel test execution. Future versions of Gradle will improve the granularity of the work in progress.
-
 The following portion of the console output shows the work in-progress display with 8 concurrent workers:
 
```

Comment:
That is correct. Only JVM tests are supported. Native does not have a parallel test execution feature. I guess we can spell it out though I am not sure if we'd need to mention Kotlin specifically here as it is an external plugin.